GOVERNANCE.md

```diff
@@ -1,45 +1,117 @@
+
+- If a CI tool is broken, they may override the tool to still submit the change.
+- Minor typos or fixes for broken tests.
+- The change was approved through other means than the standard process.
+
+#### How to become a Maintainer
+
+Contributors wanting to become maintainers are expected to:
+
+- Enable and promote successful adoptions on the user or ecosystem end
+- Actively Propose features to TiKV that are later implemented with great significance to the project direction and the eco-system
+- Able to promote substained collaboration within the project
+- Demonstrate a deep and comprehensive understanding of TiKV's codebase, technical goals, and directions
+- Handle complex problems in the code implementation process
+
+A new Maintainer must be self-nominated or nominated by an existing Maintainer by updating [TiKV Maintainers](https://github.com/tikv/tikv/blob/master/MAINTAINERS.md#the-tikv-maintainers). This triggers a voting process that that requires supermajority votes from the current Maintainers.
+
+If a Maintainer is no longer interested or cannot perform the Maintainer duties listed above, they should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of the maintainers per the voting process below.
+
+### SIGs
+
+The TiKV project is organized primarily into Special Interest Groups, or SIGs. Each SIG consists of contributors with a common purpose of advancing the TiKV project for a specific topic, such as Coprocessor, Transaction, or Scheduling. The goal of an SIG is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors. Every identifiable subpart of the project (e.g., repository, subdirectory, API, test, issue, PR) is intended to be owned by the corresponding SIG.
+
+Each SIG must have a charter that specifies its scope (topics, code repositories, and directories), responsibilities, areas of authority, how members and roles of authority/leadership are selected/granted, etc. See the [SIG charter template](/sig-governance/SIG-CHARTER-TEMPLATE.md) for details on how charters are formed and managed. SIGs should be relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by this Governance and [Sig Governance](/committee/sig-governance/sig-governance.md).
+
+SIGs are organized and operated by SIG Leads, which is a SIG internal role that oversees the health and sustained development of the SIG. Upon the initial establishment of a SIG, the maintainers will assign 2-3 Tech Leads.
+
+#### SIG Roles
+
+Within a SIG, you could find your path of contribution and growth through multiple roles - Contributor, Active Contributor, Reviewer, and Committer, each with corresponding responsiblities and requirements, as listed below:
```

Comment:
Could we merge reviewer and committer? I think there is not much difference between them

Reply:
There are some consistent settings with the TiDB community side, based on which they are building  this contributor credit system. Plus,  the difference between Committer is they have different  privileges. Committer is the highest level within a SIG. Let's just leave this as is for now.